index.js

```diff
@@ -228,8 +228,8 @@ module.exports = function (dir, optionalId, optionalName, cfg, extEvents) {
 
                 // If following were not copied/linked from template, copy from stock app hello world
                 // TODO: get stock package.json if template does not contain package.json;
-                copyContentsIfNotExists(stockAssetPath('www'), path.join(dir, 'www'));
-                copyContentsIfNotExists(stockAssetPath('hooks'), path.join(dir, 'hooks'));
+                copyIfNotExists(stockAssetPath('www'), path.join(dir, 'www'));
+                copyIfNotExists(stockAssetPath('hooks'), path.join(dir, 'hooks'));
```

Comment:
I wanted to make it more clear about what this function actually does? Wonder if we should just remove the separate function altogether?

Reply:
Well, the function would also work for plain files. But for that case the name makes no sense.

I'd keep it for now. It will have to be changed anyway in case some of apache/cordova-discuss#89 is accepted.